x-pack/platform/plugins/shared/ai_infra/product_doc_base/server/services/package_installer/steps/fetch_artifact_versions.test.ts

```diff
@@ -41,6 +44,7 @@ const createResponse = ({
 };
 
 const artifactRepositoryUrl = 'https://lost.com';
+const localArtifactRepositoryUrl = 'file://usr/local/file.xml';
```

Comment:
Should we also add a short test for windows' path? `file:///C:/path`

Reply:
added in https://github.com/elastic/kibana/pull/217046/commits/a47a1e1880f924446b33ec67ecb59a1c8a4d3cba